### PR TITLE
fix(planning): close dependency and acceptance gaps

### PR DESCRIPTION
**Pull request by pikax**
The plan could certify project-dependent checker work before project convergence, and its memory and finalization charters lacked concrete aggregate budgets and execution criteria. This amendment closes those dependency and acceptance gaps while preserving every implemented charter and existing ledger status.

- Add pending memory-budget, aggregate-admission and shipped-build verification owners; strengthen storage, churn, performance and final architecture acceptance.
- Require project snapshot and consumer convergence contracts through explicit DAG paths, guarded by a schema-checked contract inventory and regression tests.
- Bring the observation harness forward, normalize pending contract/performance obligations, reconcile successor seams, and assign release inclusion to project, source-module, bundler and framework-profile work.
- Separate consumer, internal-transit and public TypeInfo ownership; bind internal-convergence acceptance to native/TypeScript builds and nonempty public TypeInfo test execution.
- Add a read-only product acceptance report and document automatic cumulative checkpoints plus review of the actual final candidate.
- Refresh the existing closure instrument from executed graph, ledger-suite and negative-control evidence without weakening its claims.

Validation: strict DAG validation (411 nodes, 1,044 edges); 30 implementation-ledger tests; 14 ledger tests; 38 closure-instrument tests including replayed negative controls; closure view freshness; docs build; GitHub control-plane check. Independent adversarial, conformance and evidence-specialist reviews passed after fixes. The Rust runtime gate was not needed for this plan/tooling amendment; runtime obligations added here remain pending.

Related manager execution changes are local to the separate verter-dag-manager repository and are not included in this PR.

### CODE REVIEW

**Comment by pikax** [comment]
Addressed the review findings in the follow-up commit:

- Performance clauses name the applicable memory budget, allocation metrics and pre-measurement replacement SLOs. Product-report documentation now says implemented nodes.
- Contract inventories require complete populations and concrete owner nodes, paths and receiving acceptance criteria. Their validators remain pending contract deliverables. Convergence verifiers instead require already completed predecessor ownership and reject future deletion deferrals.
- Exact predecessor prose matches the project and memory dependencies. Finalization explicitly covers warmup caps, failed construction, owning contracts, complete summaries and actual CI/release execution.
- The provisioning contract now owns schemas and fixtures only. The existing production validation owner supplies types, validation, receipt caching and adapter migration; optional installation/shipping consumers depend on it without gating the required activation path.

The shipped-build negative control deliberately plants a regression that moves a required state change inside `debug_assert!`. An unconditional assertion detects the missing state change when debug assertions are disabled. Making the planted operation unconditional would stop testing that regression; the charter now states the intended control precisely.

Validation passed: strict graph (411 nodes, 1,044 edges), 30 ledger-model tests, closure-view freshness, documentation build and independent conformance rereview. Existing implementation statuses and implemented charters remain unchanged.

**Comment by pikax** [comment]
Addressed the valid follow-up findings in the pending roadmap contracts:

- EPR1 now names its ownership-inventory TOML, schema and executable validator. Its acceptance requires complete runtime outcome/consumer/displaced-route coverage and negative fixtures for omitted members, unknown or pathless owners, missing receiving acceptance IDs and conflicting bindings.
- L4 now owns a concrete future architecture-closure schema and validator, with negative controls for missing/pending required owner or ancestor rows even when L4 itself is marked implemented. The required populations explicitly include cumulative train reviews, unsupported/partial capability cases, and performance/memory results; omitted or failing required rows reject acceptance.
- E1, E2 and E3 now have distinct ownership descriptions and deletion populations for consumer migration, internal transit convergence and public DTO/graph-export acceptance.
- CAT0's transferred source-plan text now consistently assigns production migration/deletion to its later implementation owners while CAT0 produces the contract and fixtures.

This PR is an amendment to pending work, not an implementation of EPR1 or L4. The named schemas, validators and runtime evidence must be delivered by those nodes before their eventual acceptance. The existing TypeScript-mapper-specific closure instrument cannot be used as L4 acceptance. `deriveState` and the trusted ledger's intentionally status-only completion semantics remain unchanged; the future L4 validator is acceptance evidence, not another lifecycle or locator-validation authority.

Validation: strict DAG check remains 411 nodes and 1,044 edges; all 30 implementation-ledger tests pass; closure-instrument validation/freshness passes; docs build and independent review pass. Existing ledger statuses and implemented historical charters remain unchanged.

**Comment by pikax** [comment]
Addressed both remaining findings:

- EPR1 and CAT0 now explicitly require production-authority rejection and a negative fixture using a known contract-only/documentation-only owner while keeping the node, dependency path and receiving acceptance ID valid. The verification inventories and owning charter-quality contract carry the same requirement; advisory LOC budgets alone do not establish production authority.
- E2's mutation surfaces are now residual internal TypeExpr transit/reconstruction callers in `verter_session`, with current `ProjectSemanticDispatch` conversion paths as investigation anchors. Public `TypeInfoRequest`, `TypeInfoResult` and `SemanticTypeGraph` boundaries are verification-only in E2; DTO/wire/graph-export changes remain E3-owned. Legitimate authored-input lowering and terminal output materialization are explicitly preserved.

These are pending charter acceptance corrections. Strict DAG validation, closure-instrument validation/freshness, docs build and fresh independent review pass. The graph remains 411 nodes and 1,044 edges, with no existing implementation status or historical implemented charter changed.

**Comment by pikax** [comment]
Addressed the latest evidence-list findings:

- CAT0's report now explicitly includes all AC1 rejection checks, including the unknown-owner fixture.
- E2 now selects a dedicated `typeinfo-domain` gate profile in both DAG metadata and its charter. The profile runs the existing core gate, builds the native binding and TypeScript packages, then runs the TypeInfo suite using `pnpm --filter @verter/typeinfo exec vitest run --passWithNoTests=false`. The charter also requires nonzero executed tests, complete summaries and no unexpected skips.

The package test script currently permits an empty selection. I reproduced its exit-0 result with a nonexistent test filter, then verified the direct Vitest invocation exits 1 specifically because no test files were selected. Appending the opposite flag to the package script produces a duplicate-option error, which is why the profile invokes the same package's Vitest configuration directly.

Strict DAG validation, 30 implementation-ledger tests, closure-instrument validation/freshness, docs build and fresh independent review pass. This amendment binds E2's eventual execution; it does not claim that E2's runtime or its full native TypeInfo suite has been implemented/executed by this documentation update.

**Comment by pikax** [comment]
@coderabbitai review

The latest review of be5564950 completed with no actionable comments. This final correction only removes the PR's proposed performance-wording edit to D3I: origin/main has since recorded D3I as implemented, so its charter is restored byte-for-byte from the base branch and excluded from this pending-work amendment. No implementation status is changed by this PR, and strict DAG validation still passes at 411 nodes and 1,044 edges. Please review the final delta; automatic reviews were paused after the recent fix commits.

File: roadmap/0.1.0-tama/charters/compiler-compiler-perf/CPER2.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Shared compiler physical-execution and zero-work terminal. The current owner is
 
 Preflight evidence selection: preserve all four acceptance outcomes below, then select the smallest evidence set that actually discriminates the touched contract. Existing behavioral coverage, compiler/type/capability enforcement, static validation, canonical gates, bounded inspection, and benchmarks are valid when accompanied by a terse rationale.
 
-- **CPER2-AC1 — sole-owner outcome:** the named final owner must be sole and every displaced route named below must be deleted or structurally rejected. Prefer existing type, capability, dependency, compiler, or static enforcement. Add a negative or mutation test only for a plausible critical fail-closed/correctness boundary or a reproduced defect that existing evidence does not discriminate.
+- **CPER2-AC1 — ownership contract:** every inventoried outcome, consumer and displaced route has exactly one declared implementation/deletion owner. Validate contract artifacts and their negative controls; production deletion is acceptance of the later owner, not this documentation-only node.
```
[the remark this answers is not in the document]
Comment (pikax) [reply roadmap/0.1.0-tama/charters/compiler-compiler-perf/CPER2.md]: Addressed in 26f75af1f. The four ownership-contract charters require complete inventory bindings to concrete DAG nodes, dependency paths and receiving acceptance IDs, with omission/invalid-binding guards. CPER2 is a convergence verifier, so its corrected AC1 instead requires already implemented ancestor ownership and executed deletion evidence; a future owner cannot satisfy it. This distinction is required by successor-charter-quality.md and was independently re-reviewed.

File: roadmap/0.1.0-tama/charters/expansion-native-checker/NCK8.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!-- unified-charter-v2
 id=NCK8
 name=Native checker terminal and displaced-authority deletion
-predecessors=NCK7,NCKF0,PER0,UAO0,UAP0,BR0
+predecessors=NCK7,NCKF0,PER0,UAO0,UAP0,BR0,PM4
```
[the remark this answers is not in the document]
Comment (pikax) [reply roadmap/0.1.0-tama/charters/expansion-native-checker/NCK8.md:4]: Addressed in 26f75af1f. NCK8 now has the PM4 implemented-ledger entry under Exact predecessor contracts, and NCF-MP-AUG, NCF-MP-MODULE and NCF-MP-PROJECT each have the PM3 entry in that section. They match the DAG/header declarations and preserve the locator-only semantics. Strict validation passes.

File: roadmap/0.1.0-tama/charters/rev11-finalization/L4.md [hunk 3]
```diff
@@ -37,67 +37,25 @@ rescope_unrelated_packages=3
...
 
-## Targeted verification
+## Acceptance and discriminating evidence
 
-1. `cargo nextest run -p verter_bench`
-2. Run every final command in the bound `targeted-domain` profile on the squashed review candidate; targeted success alone is iteration evidence, not acceptance.
-3. Bind the preflight evidence selection and terse rationale in the review report. Behavioral code changes require TDD with a failing discriminating regression before production changes; do not invent a test or mutation solely to populate evidence.
+- **L4-AC1 — sole-owner outcome:** derive the required outcome inventory from current Rev11 DAG ancestry, including compiler-bridge ancestors, charters, owning contracts and amendments. Validate the inventory against all those sources so a contract-only outcome cannot disappear. Map every outcome to the sole final owner, consumers, displaced-route disposition and actual evidence. Explicitly classify optional and historical rows; no intended outcome disappears.
```
[the remark this answers is not in the document]
Comment (pikax) [reply roadmap/0.1.0-tama/charters/rev11-finalization/L4.md]: Addressed as a pending-node acceptance obligation in 9effd00fe. L4-AC1 and resource-and-finalization.md now require its executable closure validator to reject absent/pending required owners or transitive ancestors even if L4 itself is COMPLETE, with explicit negative fixtures. This PR amends the pending roadmap; it does not implement L4 early or change the trusted ledger/deriveState semantics. The existing TypeScript-mapper-specific validator is explicitly not L4 acceptance.
